apps/rush-lib/src/data/RushConfiguration.ts

```diff
@@ -117,6 +121,7 @@ export default class RushConfiguration {
   private _changesFolder: string;
   private _commonFolder: string;
   private _commonTempFolder: string;
+  private _commonTempFolderOverride: string | undefined;
```

Comment:
>string | undefined; [](start = 37, length = 19)

I think you should just override _commonTempFolder rather than creating a second variable. notice that this is string | undefined. It doesn't follow the pattern of the other variables, which probably tells you it shouldn't be stored here.